copy_proto_test.go

```diff
@@ -191,6 +191,37 @@ func TestStructToStruct_MaskInverse(t *testing.T) {
 	assert.Equal(t, userSrc, userDst)
 }
 
+func TestStructToStruct_MaskInverseFromMask(t *testing.T) {
+	userSrc := &testproto.User{
+		Id:          1,
+		Username:    "username",
+		Role:        testproto.Role_ADMIN,
+		Meta:        map[string]string{"foo": "bar"},
+		Deactivated: false,
+		Permissions: []testproto.Permission{testproto.Permission_EXECUTE},
+		Name:        &testproto.User_FemaleName{FemaleName: "Dana"},
+		Friends: []*testproto.User{
+			{Id: 2, Username: "friend1"},
+			{Id: 3, Username: "friend2"},
+		},
+	}
+	userDst := &testproto.User{}
+	// Mask to MaskInverse
+	mask := fieldmask_utils.MaskInverse(fieldmask_utils.Mask{"Id": fieldmask_utils.Mask{}, "Friends": fieldmask_utils.Mask{"Username": fieldmask_utils.Mask{}}})
```

Comment:
This is equivalent to 
```
mask := fieldmask_utils.MaskInverse{"Id": fieldmask_utils.Mask{}, "Friends": fieldmask_utils.Mask{"Username": fieldmask_utils.Mask{}}}
```
If so, it is expected that `Id` is _not_ copied and only `Friends.Username` are copied.
So the test case is invalid. 
Please, take a look at https://github.com/mennanov/fieldmask-utils/pull/6

Reply:
sorry for mistake, pr #6 should revert for bad test case.
@mennanov  PTAL